integration_test/myxql/storage_test.exs

```diff
@@ -159,4 +159,27 @@ defmodule Ecto.Integration.StorageTest do
     |> Enum.reject(&String.contains?(&1, "completed on"))
     |> Enum.join("\n")
   end
+
+  test "structure dump_cmd" do
+    num = @base_migration + System.unique_integer([:positive])
+    :ok = Ecto.Migrator.up(PoolRepo, num, Migration, log: false)
+
+    assert {output, 0} =
+             Ecto.Adapters.MyXQL.dump_cmd(
+               PoolRepo,
+               ["--no-create-info", "--tables", "schema_migrations"],
+               []
+             )
+
+    assert output =~ "INSERT INTO `schema_migrations` VALUES ("
```

Comment:
@josevalim I did not change anything with how the migrations work. I just took this as an example since I knew how to call it from the PR before.